docs/api/saved-objects/bulk_get.asciidoc

```diff
@@ -0,0 +1,81 @@
+[[saved-objects-api-bulk-get]]
+=== Bulk Get Objects
+
+experimental[This functionality is *experimental* and may be changed or removed completely in a future release.]
+
+The bulk-get saved object API enables you to retrieve multiple Kibana saved
+objects by id.
+
+==== Request
+
+`POST /api/saved_objects/_bulk_get`
+
+==== Request Body
+
+The request body must be a JSON array containing objects, each of which
+contains the following properties:
+
+`type` (required)::
+  (string) Valid options, include: `visualization`, `dashboard`, `search`, `config`, and `timelion-sheet`
+
```

Comment:
Should `index-pattern` be in that list of valid options?  I'm hoping it would.

Does `type` have to be a required field?  This means I can't do one _bulk_get for everything in my Kibana index :-(

Reply:
Yeah, `index-pattern` is valid. I'll add it everywhere we have a list like this.